app/dao/notifications_dao.py

```diff
@@ -353,11 +354,11 @@ def dao_timeout_notifications(timeout_period_in_seconds):
     updated = db.session.query(Notification). \
         filter(Notification.created_at < (datetime.utcnow() - timedelta(seconds=timeout_period_in_seconds))). \
         filter(Notification.status.in_([NOTIFICATION_CREATED, NOTIFICATION_SENDING, NOTIFICATION_PENDING])). \
```

Comment:
Agreed. If the notification is still in sending or pending that should mean we sent it to the provider and we did not get a delivery receipt. Is that a technical-failure or temporary-failure? 
I think that's a temporary-failure. @minglis do you have an opinion?